public/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Mock Real Estate Platform</title>
+  <link rel="stylesheet" href="css/house.css" />
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.3.3/backbone-min.js"></script>
+  <script src="/js/templates/templates.js"></script>
+  <script src="/js/main.bundled.js"></script>
+	 
+</head>
+<body>
+	<div class="center" id="header">
+		<h1>For Sale By Owner</h1>
+		<h2>A Mock Real Estate Platform</h2>
+		<h3>powered by<h3>
+		<img id="bkat" src="img/bkat.jpg" alt="bKat" style="height:50px;"><br><br>
+	</div>
+	<div class="center" id="choices">
+		<a href="/api/houses" class="button" id="allHouses">View ALL Homes for Sale</a>	
```

Comment:
I would avoid confusing your api routes with your client-side routes. A client-side route should be able to load the application up so that it displays in the same state it appeared in if you had loaded the application from the root then navigated to this specific route. In this case, you're using the routes you've defined in your express router, which are really api endpoints meant to be used for ajax calls to retrieve or update data. If you appRouter.navigate to /api/houses, then refresh the browser, it simply calls your router endpoint and tries to display the resulting data in the browser. Instead, use a route like /houses. To facilitate the ability to load routes like this and not have the server complain, look at how I used the spa-server-route in the food-truck-app index.js. You're welcome to use the same module to make this work for your application. The key takeaway is that your client-side routing needs to work if you load the routes to start with - this is one of the primary responsibilities of the Backbone Router.